aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -354,27 +358,123 @@ SparseTensor& add_out_sparse_cpu(SparseTensor& r, const SparseTensor& t, const S
 // add(Tensor, SparseTensor, Scalar)
 //    formerly known as spcadd
 // --------------------------------------------------------------------
-
 template <typename scalar_t>
-void add_dense_sparse_worker_cpu(Tensor& r, Scalar value, const SparseTensor& sparse, const Tensor& indices, const Tensor& values) {
+void add_dense_sparse_worker_scalar_cpu(Tensor& r, Scalar value, const SparseTensor& sparse, const Tensor& indices, const Tensor& values) {
```

Comment:
`add_dense_sparse_worker_scalar_cpu` scalar here is bit confusing, maybe better name it `add_dense_sparse_worker_non_hybrid_cpu`

Reply:
will change this name to add_dense_sparse_worker_non_hybrid_cpu, thanks